### ISSUE TITLE
包含webcache 字段被苹果审核拒了，有人遇见过吗

### ISSUE THREAD

**Issue by Zhangyunjiang123** [rs/SDWebImage#2202]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 3.7.6
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by bpoplauschi** [comment]
@Zhangyunjiang123 please fill in the info required by the issue template. English would be nice :) even though I think @dreampiggy speaks Chinese.

**Comment by Zhangyunjiang123** [comment]
I've solved this problem! But thank you very much for your reply.




| |
张云江
邮箱：18798799912@163.com
|

签名由 网易邮箱大师 定制

在2018年04月24日 21:17，Bogdan Poplauschi 写道：

@Zhangyunjiang123 please fill in the info required by the issue template. English would be nice :) even though I think @dreampiggy speaks Chinese.

**Comment by dreampiggy** [comment]
😅 So...I just close this issue.

I can translate this for the non-Chinese speaker, this issue is about that:
> The application which binary contains `WebCache` was rejected by Apple during App Store submitting.

But however, our company's application using SDWebImage very well. So I don't know anything about this. However, since it's soleved, just close it.
